File: javascript/calculator.js
const readline = require('readline').createInterface({
    input: process.stdin, output: process.stdout
});
readline.question('Enter expression: ', input => {
    let [a, op, b] = input.split(' ');
    a = parseFloat(a); b = parseFloat(b);
    let result;
    switch(op) {
        case '+': result = a + b; break;
        case '-': result = a - b; break;
        case '*': result = a * b; break;
        case '/': result = b !== 0 ? a / b : 0; break;
        default: result = 'Invalid operator';
    }
    console.log(result);
    readline.close();
});

File: cpp/calculator.cpp
#include <iostream>
using namespace std;

int main() {
    double a, b;
    char op;
    cout << "Enter expression (e.g., 3 + 5): ";
    cin >> a >> op >> b;
    switch(op) {
        case '+': cout << a + b; break;
        case '-': cout << a - b; break;
        case '*': cout << a * b; break;
        case '/': cout << (b != 0 ? a / b : 0); break;
        default: cout << "Invalid operator";
    }
    return 0;
}
